registry-scanner/pkg/registry/registry_test.go

```diff
@@ -0,0 +1,157 @@
+package registry
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	//nolint:staticcheck
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// Test relies on image package which is not available yet. Will uncomment as soon as it is available.
+// func Test_GetTags(t *testing.T) {
```

Comment:
Test relies on image package which is not available yet. Will uncomment as soon as it is available.